e2e/tests/childSolicitors_test.js

```diff
@@ -92,9 +128,27 @@ Scenario('HMCTS assign a different solicitor for some of the children', async ({
   caseViewPage.selectTab(caseViewPage.tabs.casePeople);
 
   for (const [index, child] of children.entries()) {
-    const solicitor = index === childWithDifferentRegisteredSolicitorIdx ? alternativeSolicitor : (index === childWithUnregisteredSolicitorIdx ? unregisteredSolicitor : mainSolicitor);
+    const solicitor = index === childWithDifferentRegisteredSolicitorIdx ? solicitor2 : (index === childWithUnregisteredSolicitorIdx ? unregisteredSolicitor : solicitor1);
     assertChild(I, index + 1, child.value, solicitor);
   }
+
+  caseViewPage.selectTab(caseViewPage.tabs.changeOfRepresentatives);
+  assertChangeOfRepresentative(I, 1, 'FPL', `${children[childWithDifferentRegisteredSolicitorIdx].value.party.firstName} ${children[childWithDifferentRegisteredSolicitorIdx].value.party.lastName}`, 'HMCTS', {removedUser: solicitor1.details, addedUser: solicitor2.details });
+  assertChangeOfRepresentative(I, 2, 'FPL', `${children[childWithUnregisteredSolicitorIdx].value.party.firstName} ${children[childWithUnregisteredSolicitorIdx].value.party.lastName}`, 'HMCTS', {removedUser: solicitor1.details, addedUser: unregisteredSolicitor.details });
+});
+
+Scenario('Solicitor can request representation via NOC only after case submission and Cafcass solicitor is set', async ({I, caseListPage, noticeOfChangePage, caseViewPage}) => {
+  await setupScenario(I);
+  await I.signIn(solicitor3);
+  caseListPage.verifyCaseIsNotAccessible(caseId);
+
+  await noticeOfChangePage.userCompletesNoC(caseId, 'Swansea City Council', children[11].value.party.firstName, children[11].value.party.lastName);
+
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  assertChild(I, 12, children[11].value, solicitor3);
```

Comment:
same here, waiting for the tab to load is probably a good idea for stability (might make most sense in `assertChild(...)`)